src/current/_includes/v23.1/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -59,12 +59,10 @@ The `cockroach` process listens on `127.0.0.1:26257` and `localhost:26257`, and
       --env COCKROACH_PASSWORD={PASSWORD} \
       --name=roach-single \
       --hostname=roach-single \
-      --net=roachnet \
```

Comment:
Thanks, leaving the network details in the single-node example was a mistake on my part.

Reply:
You're welcome!